### ISSUE TITLE
[Azure Tools Storage Explorer - Platform]Update Accessibility Conformance Documentation

### ISSUE THREAD

**Issue by ManishaC1** [microsoft/AzureStorageExplorer#5735]
**Github Tags**
#A11yAAT; #A11yMAS; #A11yTCS; #Azure Tools Storage Explorer; #BM_Azure Tools Storage Explorer_May2022; #Benchmark; #DesktopApp; #FTP; #Win11; #A11ySev3;

## Description: 
This bug is to ensure that an AAT entry is created/updated with most recent test results. This would ensure that conformance documentation up to date.

**Product Name**: Azure Tools Storage Explorer
 STID:5e4cbb05-3006-478c-a881-3a2c1769efd5

## Recommendation: 
Fix and close other bugs before starting accessibility conformance documentation.  Any bug that will not be fixed and closed within SLA must be declared in the AAT.  The SLA for this bug is also 60 days.

## Link to the AAT entry:
https://cela-ra-aat-prod.azurewebsites.net/#/workflow/9ef42ea4-422c-42b5-b4e9-d6eb49a2e971

## Steps to Navigate:

1. Hit on the URL https://cela-ra-aat-prod.azurewebsites.net/#/home
2. Hit enter on the "My work environment" button.
3. Tab to navigate to "Find Products" tab.
4. Tab to navigate to "Service tree ID" field.
5. Paste your service tree id "5e4cbb05-3006-478c-a881-3a2c1769efd5" and hit on "GO".
6. Search for the latest AAT draft entry based on platform.
[How to Guide - Creating & Updating/re-publishing Conformance documentation (sharepoint.com)](https://microsoft.sharepoint.com/teams/cea11y/SitePages/How-to-Guide---Creating-Conformance-documentation.aspx?OR=Teams-HL&CT=1634788203040)

When the AAT entry is completed, please resolve the bug with the AAT URL. Trusted tester will review the AAT and send comments if any. Once this review is completed, then mark the AAT entry for review in the AAT itself - this will send the entry to CELA for their review. When the AAT is in Reviewed/Finalized/Public status, this bug will be Closed.
France Reporting Reqs" section:
Please refer to [ "French Decree Scope for Websites" page ](https://microsoft.sharepoint.com/teams/cea11y/SitePages/French-Decree-Scope-for-Websites%2c-mobile-&-web-applications%2c-and-online-services%281%29.aspx?xsdata=MDN8MDF8fDg1YTVmZGUyYzNlYTRhM2U5ZDMwYTMxNDdiYzgwNjFhfDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDF8MHw2Mzc2ODA0MTE0MDQ1NzU1OTB8R29vZHxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaUlpd2lRVTRpT2lJaUxDSlhWQ0k2TVRKOQ%3D%3D&sdata=TmxFSWN4MklPR1ZuMDZ2M0UrZ2lUR1JnV2svWHVSd093ZzRDNkdoWEhyaz0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-stfe%40microsoft.com&OR=Teams-HL&CT=1637334514141&sourceId=&params=%7b%22AppName%22:%22Teams-Desktop%22%2c%22AppVersion%22:%2227%2f21110108713%22%7d) to determine if the product is in scope for French decree or not.  If it is in scope, verify that the French Reporting Reqs section of the AAT is filled in (for questions on what and how to verify content in this section, please contact Amita Singh (amitasingh@microsoft.com).  If it is NOT in scope, then the checkbox in the AAT that reads "To the best of my knowledge, this website, service or application does not have French reporting obligations." must be checked.

**Comment by ManishaC1** [comment]
Bug query-->https://github.com/microsoft/AzureStorageExplorer/issues?q=BM_Azure+Tools+Storage+Explorer_May2022

**Comment by ManishaC1** [comment]
Closing the AAt bug, as one should not log AAT bug when the repository is Github.
